php/login.php

```diff
@@ -0,0 +1,141 @@
+<?php
+require_once("database.php");
+
+session_start();
+if (isset($_SESSION["loggedin"]) && $_SESSION["loggedin"] === true) {
+  //needs to be point to main page after login
+  header("location: welcome.php");
+  exit;
+}
+
+$conn = connect_local();
+$email = $password = "";
+$email_err = $password_err = $login_err = "";
+
+if ($_SERVER["REQUEST_METHOD"] == "POST") {
+  if (empty(trim($_POST["email"]))) {
+    $email_err = "Please enter your email!";
+  } else {
+    $email = trim($_POST["email"]);
+  }
+  if (empty(trim($_POST["password"]))) {
+    $password_err = "Please enter your password!";
+  } else {
+    $password = trim($_POST["password"]);
+  }
+
+  if (empty($email_err) && empty($password_err)) {
+    $sql = "SELECT id, email, password FROM user WHERE email= ?";
+    $stmt = $conn->prepare($sql);
+    $param_email = trim($_POST["email"]);
+    $stmt->bindParam(1, $param_email);
+    if ($stmt->execute()) {
+      if ($stmt->rowCount() == 1) {
+        if ($row = $stmt->fetch()) {
+          $id = $row["id"];
+          $email = $row["email"];
+          //password is not yet hashed in the db
+          $hashed_password = $row["password"];
+          #if(password_verify($password, $hashed_password)){
+          if ($password === $hashed_password) {
+            session_start();
```

Comment:
We need the user_name later to adress the user. Please save it into the session.